### PR TITLE
FEAT: Added Grid component

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-02-16 13-39-31
Screenshot from 2022-02-16 13-48-26

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

File: Components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.grid-section{
+    margin: -2rem auto auto 4rem;
```
Comment (bharati-21) [inline Components/Grid/grid.css]: Any specific reasons for using `-2rem` here?

File: Components/Grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    width: 80%;
+}
+  .grid-1{
+    display: grid;
+    grid-template-columns:  1fr 1fr;
```
Comment (bharati-21) [inline Components/Grid/grid.css]: Could use shorthand repeat(2,1fr) here.

File: Components/Grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+.grid-2{
+    display: grid;
+    grid-template-columns:  1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+    grid-gap: 1rem;
+    padding: 2rem;
+}
+.grid-item{
+    margin: auto;
+    width: 100%;
```
Comment (bharati-21) [inline Components/Grid/grid.css]: Same here as well. 👍🏽

File: Components/Grid/grid.html [hunk 4]
```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (bharati-21) [inline Components/Grid/grid.html:7]: You can also add the meta author and description tags here.

File: Components/Grid/grid.html [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+                <div class="code-frame">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28184%2C176%2C141%2C0%29&t=3024-night&wt=none&l=auto&width=680&ds=true&dsyoff=0px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=4x&wm=false&code=%2520%253Cdiv%2520class%253D%2522grid-1%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522grid-item%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F200%2522%2520alt%253D%2522item-1%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522grid-item%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F200%2522%2520alt%253D%2522item-2%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522grid-item%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F200%2522%2520alt%253D%2522item-3%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522grid-item%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F200%2522%2520alt%253D%2522item-4%2522%253E%250A%2520%253C%252Fdiv%253E"
+                        style="width: 742px; height: 283px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (bharati-21) [inline Components/Grid/grid.html]: Kindly avoid inline styles.

File: Components/Grid/grid.html [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+                        style="width: 751px; height: 376px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (bharati-21) [inline Components/Grid/grid.html]: Here as well.

**Review by bharati-21 — APPROVED** [review]
Just dropped in a few suggestons. Looks good to be merged!

File: Components/Grid/grid.css (hunk 1 above)
Comment (kotesh-arya) [reply Components/Grid/grid.css]: No no, I just overlooked that and fixed it now, Thanks for pointing it out 👍🏻

File: Components/Grid/grid.css (hunk 3 above)
Comment (kotesh-arya) [reply Components/Grid/grid.css]: Yeah, I've done that 👍🏻

File: Components/Grid/grid.html (hunk 5 above)
Comment (kotesh-arya) [reply Components/Grid/grid.html]: Sure 👍🏻

File: Components/Grid/grid.html (hunk 6 above)
Comment (kotesh-arya) [reply Components/Grid/grid.html]: Yep, I have updated this 👍🏻

File: Components/Grid/grid.css (hunk 2 above)
Comment (kotesh-arya) [reply Components/Grid/grid.css]: Yep, Done 👍🏻

File: Components/Grid/grid.html (hunk 4 above)
Comment (kotesh-arya) [reply Components/Grid/grid.html:7]: Done it 👍🏻

**Comment by kotesh-arya** [comment]
Thanks For these Helpful suggestions guys, I have updated my code according to your suggestions 😃

**Pull request merged**